src/arch/x86/kernel/vspace.c

```diff
@@ -149,6 +149,17 @@ BOOT_CODE bool_t map_kernel_window_devices(pte_t *pt, uint32_t num_ioapic, paddr
     assert(idx == (PPTR_APIC & MASK(LARGE_PAGE_BITS)) >> PAGE_BITS);
     pt[idx] = pte;
     idx++;
+
+#ifdef CONFIG_VGA_PRINTING
```

Comment:
If we do this mapping last, then then mapping address of the other device does not change if this is enabled or not used. Might be more convenient for debugging.